doc/modules/cross_validation.rst

```diff
@@ -121,11 +121,10 @@ time)::
   >>> scores
   array([0.96..., 1.  ..., 0.96..., 0.96..., 1.        ])
 
-The mean score and the 95\% confidence interval of the score estimate are hence
-given by::
+The mean score and the standard deviation are hence given by::
 
-  >>> print("Accuracy: %0.2f (+/- %0.2f)" % (scores.mean(), scores.std() * 2))
-  Accuracy: 0.98 (+/- 0.03)
+  >>> print("%0.2f accuracy with a standard deviation of %0.2f" % (scores.mean(), scores.std()))
+  0.98 accuracy with a standard deviation of 0.03
```

Comment:
The `SVC` needs to be updated with a `random_state` to make the results deterministic:

```py
clf = svm.SVC(kernel='linear', C=1, random_state=42)
```

Reply:
Good point! Just fixed it.